File: Stage17/C/Problem11050_1.c
#include <stdio.h>

int main() {
    int N, K;
    scanf("%d %d", &N, &K);

    int result = 1;
    
    for (int i = 1; i <= K; i++) {
        result = result * (N - i + 1) / i;
    }

    printf("%d\n", result);

    return 0;
}

File: Stage17/Python/Problem11050_1.py
import math

def main():
    N, K = map(int, input().split())
    result = 1
    
    for i in range(1, K + 1):
        result = result * (N - i + 1) // i
    
    print(result)

if __name__ == "__main__":
    main()